code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -517,13 +547,58 @@
 	if(target in hound.module.modules)
 		return
 	if(length(contents) > (max_item_count - 1))
-		user << "<span class='warning'>Your [src.name] is full. Eject or process contents to continue.</span>"
+		to_chat(user, "<span class='warning'>Your [src.name] is full. Eject or process contents to continue.</span>")
+		return
+
+	if(istype(src,/obj/item/device/dogborg/sleeper/compactor/analyzer))
+		if(istype(target, /obj/item))
+			var/obj/target_obj = target
+			if(target_obj.w_class >= ITEMSIZE_LARGE)
+				to_chat(user, "<span class='warning'>\The [target] is too large to fit into your [src.name]</span>")
+				return
+			user.visible_message("<span class='warning'>[hound.name] is ingesting [target.name] into their [src.name].</span>", "<span class='notice'>You start ingesting [target] into your [src.name]...</span>")
+			if(do_after(user, 30, target) && length(contents) < max_item_count)
+				target.forceMove(src)
+				user.visible_message("<span class='warning'>[hound.name]'s internal analyzer groans lightly as [target.name] slips inside.</span>", "<span class='notice'>Your garbage compactor groans lightly as [target] slips inside.</span>")
+				playsound(hound, 'sound/vore/gulp.ogg', 30, 1)
+				if(istype(target,/obj/item))
+					var/obj/item/tech_item = target
+					for(var/T in tech_item.origin_tech)
+						to_chat(user, "<span class='notice'>\The [tech_item] has level [tech_item.origin_tech[T]] in [CallTechName(T)].</span>")
+				if(length(contents) > 11) //grow that tum after a certain junk amount
+					hound.sleeper_r = TRUE
+					hound.updateicon()
+				if(UI_open == TRUE)
+					sleeperUI(usr)
+			return
+
+		else if(ishuman(target))
+			var/mob/living/carbon/human/trashman = target
+			if(patient)
+				to_chat(user, "<span class='warning'>Your [src.name] is already occupied.</span>")
+				return
+			if(trashman.buckled)
+				to_chat(user, "<span class='warning'>[trashman] is buckled and can not be put into your [src.name].</span>")
+				return
+			user.visible_message("<span class='warning'>[hound.name] is ingesting [trashman] into their [src.name].</span>", "<span class='notice'>You start ingesting [trashman] into your [src.name]...</span>")
+			if(do_after(user, 30, trashman) && !patient && !trashman.buckled && length(contents) < max_item_count)
+				trashman.forceMove(src)
+				trashman.reset_view(src)
+				update_patient()
+				processing_objects.Add(src)
+				user.visible_message("<span class='warning'>[hound.name]'s internal analyzer groans lightly as [trashman] slips inside.</span>", "<span class='notice'>Your garbage compactor groans lightly as [trashman] slips inside.</span>")
```

Comment:
Psst. Still calling it a garbage compactor on the message that the borg sees.

Reply:
whoops thanks